src/cps/env.cpp

```diff
@@ -10,17 +11,31 @@
 
 namespace cps {
 
+    namespace {
+        std::vector<fs::path> get_paths(const char * input) {
+#ifdef _WIN32
+            std::vector<std::string> path_strs = utils::split(input, ";");
+#else
+            std::vector<std::string> path_strs = utils::split(input, ":");
+#endif
```

Comment:
Would it be better to do something like:
```cpp
const char * path_sep =
#ifdef _WIN32
    ";"
#else
    ":"
#endif
;
```
sot aht we don't have to repeat the calls to `utils::split`?